pkg/controller/namespace/namespace_controller.go

```diff
@@ -107,6 +107,13 @@ func (nm *NamespaceController) enqueueNamespace(obj interface{}) {
 		utilruntime.HandleError(fmt.Errorf("Couldn't get key for object %+v: %v", obj, err))
 		return
 	}
+
+	namespace := obj.(*v1.Namespace)
```

Comment:
I don't know that we need to fix it in this PR, but it seems odd that we type assert to a `*v1.Namespace` above where we set up the shared informer event handler funcs, but `enqueueNamespace` takes an `interface{}` so you're having to re-assert.